tgui/packages/tgui/interfaces/TransitShuttle.js

```diff
@@ -0,0 +1,43 @@
+import { useBackend } from '../backend';
+import { Box, Section, Divider, Button, Table, BlockQuote } from '../components';
+import { Window } from '../layouts';
+import { ReagentBlocks } from './EspressoMachine';
+
+export const TransitShuttle = (props, context) => {
+  const { act, data } = useBackend(context);
+  const Destinations = data.Destinations || [];
+  const currentlocation = data.currentlocation || [];
+
+  const {
+    shuttlename,
+    moving,
+    locked,
+  } = data;
+  return (
+    <Window height="520" width="300" title={shuttlename} >
+      <BlockQuote>the {shuttlename} is currently at {currentlocation.name}
```

Comment:
Would be nice if this updated while in transit (and also had some padding, see `Window.Content` I think?)